docs/reference/server-configuration.md

```diff
@@ -691,7 +801,7 @@ By default, any amount of lag is allowed. For using this functionality, the serv
 
 ## `ksql.query.pull.table.scan.enabled`
 
-**Per query:** yes
+**Per query:** yes (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

Comment:
@rodesai  ditto above 

```suggestion
**Per query:** yes
```

Reply:
This does not need to be settable per query. It's a feature flag. You arguably shouldn't even be able to set it using `ALTER SYSTEM`.

Reply:
Makes sense -- feature flags should be server properties, in that case @JimGalasyn this one should just be a "no"